lib/tasks/mutant.rake

```diff
@@ -0,0 +1,17 @@
+CLASSES = %w[
+  CaseRequest
+].freeze
+
+task :mutant do
+  CLASSES.each do |klass|
+    vars = 'RAILS_ENV=test NOCOVERAGE=true'
+    flags = '--use rspec --fail-fast'
+    unless system("#{vars} mutant -r ./config/environment #{flags} #{klass}")
```

Comment:
@mbj thanks for that.  We'd arrived at that working through other components in the architecture and will backport that work here when we have some time.